examples/regular_polygon.py

```diff
@@ -0,0 +1,181 @@
+            ):
+                print("\nQuitting program...")
+                pg.quit()
+                sys.exit()
+            if event.type == pg.MOUSEBUTTONDOWN:
+                if pg.mouse.get_pressed() == (1, 0, 0):
+                    self.mouse_left = not self.mouse_left
+                elif pg.mouse.get_pressed() == (0, 0, 1):
+                    self.mouse_right = True
+            if event.type == pg.MOUSEBUTTONUP:
+                if pg.mouse.get_pressed() == (0, 0, 0):
+                    self.mouse_left = False
+                    self.mouse_right = False
+            if event.type == pg.MOUSEWHEEL:
+                self.mouse_wheel = True
+                if event.y > 0:
+                    self.sides = self.sides + 1 if self.sides < 10 else 10
+                elif event.y < 0:
+                    self.sides = self.sides - 1 if self.sides > 3 else 3
+
+    def draw_polygon(
+        self,
+        num_sides=3,
+        center=(0.0, 0.0),
+        radius=1.0,
+        angle=0.0,
+        color=(255, 255, 255),
+    ):
+        """
+        Method draw_polygon: draws a regular polygon
+        """
+        box_side = 2 * radius
+        box_x, box_y = center[0] - box_side // 2, center[1] - box_side // 2
+        box = pg.surface.Surface((box_side, box_side)).convert_alpha()
+        box.fill([0, 0, 0, 0])
+        polygon = geometry.regular_polygon(
+            num_sides, (box_side // 2, box_side // 2), radius, angle
+        )
+
+        pg.draw.polygon(box, color, polygon.vertices)
+        self.screen.blit(box, (box_x, box_y))
+
+    def draw_text(self, text, color, pos):
+        """
+        Method draw_text: draws text
+        """
+        text_surface = self.font.render(text, True, color)
+        text_rect = text_surface.get_rect(center=pos)
+        text_rect.center = pos
+        text_bg = pg.surface.Surface((80, text_surface.get_height() + 5))
+        text_bg_size = text_bg.get_size()
+        text_bg_rect = text_bg.get_rect(center=pos)
+        # pos - (text_bg_size // 2)
+        text_bg_rect.center = tuple((p - tbg // 2) for p, tbg in zip(pos, text_bg_size))
+        text_bg.fill(GRAY_BACKGROUND)
+        self.screen.blit(text_bg, text_bg_rect.center)
+        self.screen.blit(text_surface, text_rect)
+
+    def draw(self):
+        """
+        Method draw: draws scene
+        """
+        # Mouse events
+        if self.mouse_left:
+            angle = math.tau * random.uniform(0.0, 1.0)
+            x = pg.mouse.get_pos()[0] + round(math.cos(angle) * self.radius * 0.5)
+            y = pg.mouse.get_pos()[1] + round(math.sin(angle) * self.radius * 0.5)
+            color = (
+                random.randint(0, 255),
+                random.randint(0, 255),
+                random.randint(0, 255),
+            )
+            self.draw_polygon(
+                num_sides=self.sides,
+                center=(x, y),
+                radius=self.radius,
+                angle=33.33333,
```

Comment:
maybe pass it a random angle?
```suggestion
                angle=random.randint(0, 360),
```